pymc/distributions/shape_utils.py

```diff
@@ -472,8 +472,8 @@ def convert_size(size: Size) -> Optional[StrongSize]:
     if size is None:
         return None
 
-    if isinstance(size, int) or (isinstance(size, TensorVariable) and size.ndim == 0):
-        size = (size,)
+    if isinstance(size, int) or (isinstance(size, TensorVariable) and size.ndim==1 or size.ndim==0):
+        size = tuple(size,)
     elif isinstance(size, (list, tuple)):
         size = tuple(size)
     else:
```

Comment:
Does this work? I don't think we have to do anything if it is already a TensorVariable. We could assert it has `ndim < 2`. but that's about it.

```suggestion
    if isinstance(size, int):
        size = tuple(size,)
    elif isinstance(size, (list, tuple)):
        size = tuple(size)
    elif not isinstance(TensorVariable):
```